assets/app/scripts/controllers/create.js

```diff
@@ -94,6 +100,132 @@ angular.module('openshiftConsole')
       $scope.filter.tag = tag;
     };
 
+    $scope.aceLoaded = function(editor) {
+      var session = editor.getSession();
+      session.setOption('tabSize', 2);
+      session.setOption('useSoftTabs', true);
+      editor.setDragDelay = 0;
+      editor.$blockScrolling = Infinity;
+
+      $('.from-file .editor').animate({
+        height: Math.floor(window.innerHeight * 0.50) + 'px'
+      }, 30, function() {
+        editor.resize();
+      });
+    };
+
+    $scope.create = function() {
+      delete $scope.alerts['create'];
+      delete $scope.alerts['parsing'];
+      var resource;
+      if ($scope.editor.fileExtension === "yaml") {
+        try {
+          resource = YAML.parse($scope.editor.content);
+        } catch (e) {
+          $scope.alerts['parsing'] = {
+            type: "error",
+            message: "An error occurred during parsing editors content.",
+            details: "Reason: " + $filter('getErrorDetails')(e)
+          };
+          return;
+        }
+      } else if ($scope.editor.fileExtension === "json") {
+        try {
+          resource = JSON.parse($scope.editor.content);
+        } catch (e) {
+          $scope.alerts['parsing'] = {
+            type: "error",
+            message: "An error occurred during parsing editors content.",
+            details: "Reason: " + $filter('getErrorDetails')(e)
```

Comment:
same thing except a JSON parse exception is probably going to be a different format than the yaml parse exception, just try putting in bad JSON and see what the exception looks like